markdown/usage/troubleshooting.md

```diff
@@ -3,34 +3,99 @@ title: Troubleshooting
 subtitle: How to troubleshoot common mistakes and issues
 ---
 
-* [Input files not found](#input-files-not-found)
-* [Data organization](#data-organization)
-* [Image cannot be built](#image-cannot-be-built)
-* [Cannot find input files when using Singularity](#cannot-find-input-files-when-using-singularity)
-* [Warning about 'sticked on revision'](#warning-about-sticked-on-revision)
-* [Extra resources and getting help](#extra-resources-and-getting-help)
+<!-- TOC -->
+
+- [Input files not found](#input-files-not-found)
+  - [Direct input](#direct-input)
+  - [Sample sheet input](#sample-sheet-input)
+  - [Output for only a single sample although I specified multiple with wildcards](#output-for-only-a-single-sample-although-i-specified-multiple-with-wildcards)
+  - [Data organization](#data-organization)
+- [The pipeline crashes almost immediately with an early pipeline step](#the-pipeline-crashes-almost-immediately-with-an-early-pipeline-step)
+  - [Error related to Docker](#error-related-to-docker)
+  - [Error related to Singularity](#error-related-to-singularity)
+- [Cannot find input files when using Singularity](#cannot-find-input-files-when-using-singularity)
+- [Warning about sticked on revision](#warning-about-sticked-on-revision)
+- [I get a exceeded job memory limit error](#i-get-a-exceeded-job-memory-limit-error)
+- [Crashed pipeline with an error but Nextflow is still running](#crashed-pipeline-with-an-error-but-nextflow-is-still-running)
+  - [A step of a pipeline wasn't executed](#a-step-of-a-pipeline-wasnt-executed)
+- [My pipeline update doesn't seem to do anything](#my-pipeline-update-doesnt-seem-to-do-anything)
+- [Unable to acquire lock error](#unable-to-acquire-lock-error)
+- [Extra resources and getting help](#extra-resources-and-getting-help)
+
+<!-- /TOC -->
 
 ## Input files not found
 
-If only no file, only one input file , or only read one and not read two is picked up then something is wrong with your input file declaration
+If the pipeline can't find your files then you will get the following error
+
+```bash
+ERROR ~ Cannot find any reads matching: *{1,2}.fastq.gz
+```
 
-1. The path must be enclosed in quotes (`'` or `"`)
+### Direct input
+
+Or when you're using a input method like `--input '/<path>/<to>/*_fq.gz'`, but only pick up one file, or only one file per pair being processed during the run, please note the following:
```

Comment:
I'll add that it should follow glob pattern matching: https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob

Reply:
cf Nextflow docs

Reply:
I did recommend using `[!bar]` and `{foo,baz}` to filter input files for sarek.